test/integration/targets/azure_rm_virtualmachine_facts/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/azure
+posix/ci/cloud/group2/azure
+posix/ci/cloud/group2/smoketest
```

Comment:
Remove this line. This test does not need to be added to the smoketest.

Reply:
Line removed.